v3.5

Trevelopment opened the pull request:
- Toggle Background Button
- List Looping
- Playback Touch controls
- Updated Readme
- Move Changelog to its own file
- logFile as a variable
- Some other small fixes
- Some more log entries.
- Save Recently Played List and Background Toggle Button State

Trevelopment committed:
Merge pull request #5 from vicbam85/master

previousVideoTrack changed to a local var

Trevelopment committed:
Bump to v2.9

previousVideoTrack changed to a local var

Trevelopment committed:
Merge remote-tracking branch 'Trevelopment/master'

Trevelopment committed:
v2.9

Bump to v2.9 for AIO v2.5.5 release

Trevelopment committed:
v2.9

Bump to v2.9 for AIO v2.5.5 release

Trevelopment committed:
v2.9

Bump to v2.9 for AIO v2.5.5 release

Trevelopment committed:
Merge

Trevelopment committed:
Toggle background

Trevelopment committed:
:thought_balloon:

Trevelopment committed:
README

Tables with Github Flavored Markdown

Trevelopment committed:
🐂

Trevelopment committed:
small adjustments

Trevelopment committed:
Minor Bug Fixes

Trevelopment committed:
Some more small fixes

Trevelopment committed:
ERR

Trevelopment committed:
VideoPlayer v2.9

Trevelopment committed:
Show Memory Error

Trevelopment committed:
VideoPlayer-v3.0

Toggle between Unicode and ASCII Mode

Trevelopment committed:
v3

Trevelopment committed:
v3.2

mahouda28 commented:
please can you just do an other app like this but it can read m3u file to play tv with m3u like iptv like vlc to see many channel in m3u with the connexion internet in mzdconnect please do you can this like this up is just the same but it read m3u file

Trevelopment committed:
Remove Unicode Mode

Trevelopment committed:
Music Player w/ MP3/FLAC Support

Trevelopment committed:
CSS fix

Trevelopment committed:
Remove Logging

Trevelopment committed:
SBNs & Audio Pause Hack

Trevelopment committed:
v3.5

  *   Created starting method StartVideoPlayerApp to start Player instead of using the document 'ready' event
  *   Repeat now toggles none - 1 - all in one button
  *   Shows gplay error in error message
  *   After hitting an error will make up to 3 attempts to re-start the video every 10 seconds
  *   Added Multicontroller 'hold' actions
  *   Video Resumes where it left off when shifting out of reverse
  *   Added to options panel "Resume Play" when checked:
  *   - Reloads saved video list on app reopen and resume (does not scan USB again)
  *   - Resumes the video where you left off if interrupted or app is exited.
  *   - If the video is stopped and you are in the list on exit you will return to the list when reopened.
  *   Multicontroller Hold events
  *   SBNs for toggles

Trevelopment committed:
Merge remote-tracking branch 'GitLAb/master'

Trevelopment committed:
Merge with Vic

Trevelopment commented:
Hey @vicbam85 Id love it if you want to test v3.5 and give me some feedback before I release it if you have some time there are many new features & fixes including yours for v3.4:
  *   Fixed issues with random stopping in random
  *   Fixed issues with trying to go back with no previous track
  *   Video title filters out the extension and turns underscores to spaces
  *   Repeat now one button toggles: none - 1 - all
  *   StatusBarNotifications for toggles
  *   Added Multicontroller 'hold' actions
  *   Shows gplay error in error message
  *   After hitting an error will make up to 3 attempts to re-start the video in 10 second intervals
  *   Video Resumes where it left off when shifting out of reverse
  *   Added to option "Resume Play" - when checked:
  *   - Resumes the video where you left off if it was interrupted or the app was exited while playing.
  *   - Saves video list to reopen and resume quickly (reloads if switch to music or press reload button)
  *   - If the video is stopped and you are in the list when you exit you will return to the list when reopened.
  *   Added "Black Out Background" option - Will overlay all other GUI layers leaving only the video and solid black background
  *   - Video Title will show centered above video and time below in the lower left corner for about 5 seconds and fade out
  *   - If "Title to Statusbar" is checked, title and time will stay visible (but can be toggled with the multicontroller).
  *   - This is ON TOP of all layers including the statusbar and bottom controls while the video player background is ON THE BOTTOM of all the other layers
  *   - Pausing the video temporarily hides the overlay

Trevelopment committed:
v3.5

Trevelopment committed:
space

Trevelopment committed:
Polishing it up

Trevelopment committed:
Some little adjustments

Trevelopment committed:
Merge remote-tracking branch 'VIC_BAM85/master'

Trevelopment committed:
Make sure after reverse we are in the _videoplayer app

Trevelopment committed:
Fix parking sensor issue

needs testing - changes make every time the _noLongerDisplayed is fired as long as the video hasnt started yet it will keep the same value insted of becoming null the 2nd time

Trevelopment committed:
use a class to determine player state & layout

Cuts down on the jquery ops and handle layouts with CSS (better performance)

Trevelopment committed:
Unmount swapfile on system shutdown

Attempts to unmount a mounted swapfile by listening for 'WaitForEnding' || 'PowerDownAnimation' contexts

Trevelopment committed:
smother retry playing video on memory error

Trevelopment committed:
Merge remote-tracking branch 'origin/master'

Trevelopment committed:
Official v3.5 Release

Trevelopment committed:
Support for v70

Trevelopment committed:
Kill AA & Carplay

Trevelopment committed:
File Path
